include/galsim/SBMoffatImpl.h

```diff
@@ -45,26 +45,26 @@ namespace galsim {
         double maxK() const;
         double stepK() const;
 
-        void getXRange(double& xmin, double& xmax, std::vector<double>& ) const 
+        void getXRange(double& xmin, double& xmax, std::vector<double>& ) const
```

Comment:
> In the meantime..., supposedly adding a ?w=1 to a github url will ignore whitespace changes

That's true, but it does mean you can no longer comment on the code (or at least I wasn't able to when I used that option).